checker/manual/creating-a-checker.tex

```diff
@@ -483,30 +489,7 @@ \subsection{Declaratively specifying implicit annotations\label{declarative-type
 automatically add that qualifier.
 
 In short, the types and trees can be
-specified via any combination of five fields in \refclass{framework/qual}{ImplicitFor}:
-
-  \begin{itemize}
-  \item \code{trees}: an array of
-    \href{\TreeAPIBase{}/tree/Tree.Kind.html?is-external=true}{\code{com.sun.source.tree.Tree.Kind}}, e.g.,
-    \code{NEW\_ARRAY} or \code{METHOD\_INVOCATION}
-  \item \code{types}: an array of
-    \refModelclass{type}{TypeKind}, e.g., \code{ARRAY}
-    or \code{BOOLEAN}
-  \item
-    \begin{sloppypar}
-    \code{treeClasses}: an array of class literals for classes
-    implementing \refTreeclass{tree}{Tree}, e.g.,
-    \code{LiteralTree.class} or \code{ExpressionTree.class}
-    \end{sloppypar}
-  \item \code{typeClasses}: an array of class literals for classes
-    implementing \code{javax.lang.model.type.TypeMirror}, e.g.,
-    \code{javax.lang.model.type.PrimitiveType}.  Often you should use
-    a subclass of \refclass{framework/type}{AnnotatedTypeMirror}.
-  \item \code{stringPatterns}: an array of regular expressions that will
-    be matched against
-    string literals, e.g., \code{"[01]+"} for a binary number.  Useful
-    for annotations that indicate the format of a string.
-  \end{itemize}
+specified via any combination of four fields in \refclass{framework/qual}{ImplicitFor}.
```

Comment:
Why did you remove all explanations instead of just removing the ones that are being removed?